src/auth/session/storage/mongodb.ts

```diff
@@ -75,6 +75,29 @@ export class MongoDBSessionStorage implements SessionStorage {
     return true;
   }
 
+  public async deleteSessions(ids: string[]): Promise<boolean> {
+    await this.ready;
+    for (const id of ids) {
+      await this.collection.deleteOne({id});
+    }
```

Comment:
Instead of looping over the entries, could we just use [`deleteMany`](https://www.mongodb.com/docs/v4.4/reference/method/db.collection.deleteMany/?_ga=2.7681010.1394317485.1657638452-2132842715.1657638452#db.collection.deletemany--)?